MiddlesexMA/build.gradle

```diff
@@ -18,7 +18,8 @@ sourceCompatibility = JavaVersion.VERSION_1_8
 targetCompatibility = JavaVersion.VERSION_1_8
 
 //TODO: Use credentials from environment variables.
-run.args = ["jaildata.csv","username","password"]
+run.args = ["/Users/kimengie/Documents/Data/Middlesex/Burlington_PD_DDJ_Data2015-2016/burlingtonPD_formatted_1-1-17.csv",
```

Comment:
it doesn't really matter, but this should probably be reverted to ["jaildata.csv","username","password"]